admin-core/assets/src/common/common.scss

```
@@ -458,3 +458,7 @@
 [dir="rtl"] .uag-dashboard-app button.bg-spectra[aria-checked="true"] span {
 	transform: translateX(20px);
 }
+.fse-trash-font {
+	fill: #eb4848;
```

Was this color provided by the designers?
Will it be possible to use the hex for Tailwind's Red 500 instead?
```suggestion
	fill: #ef4444;
```